### PR TITLE
Feature/orthographic UI

### PR DESCRIPTION
**Pull request by interim17**
Problem
=======
This closes [#279](https://github.com/simularium/simularium-website/issues/279) / [#320](https://github.com/simularium/simularium-website/issues/320) and [#419](https://github.com/simularium/simularium-website/issues/419).

Orthographic UI (#419) was to be implemented in tandem with button styling. In hindsight: review would be easier if this were two PRs, one for styling, one for implementing the camera. Luckily it's not a huge merge.

Solution
========

Camera:
Add UI buttons and props to call setCameraType. (see note about dev build below)

Styling:
Updated colors, size, spacing, and borders of camera and playback controls. Did some cleanup adding .container wrappers on CSS rules and renaming a few things.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
New viewer version released with ortho code!

npm run start
toggle orthographic/perspective modes in the camera controls
check out the styling of camera and playback controls

Screenshots (optional):
----------------
ortho
persp
-------

### CODE REVIEW

**Comment by meganrm** [comment]
the tool tips look backwards to me on these screen shots

File: src/assets/orthographic.svg
Comment (meganrm) [inline src/assets/orthographic.svg:1]: this is the perspective camera image (the box gets littler in the back)

File: src/components/CameraControls/index.tsx [hunk 1]
```diff
@@ -169,64 +215,75 @@ const CameraControls = ({
...
-                        ])}
-                        onClick={() => saveFocusMode(!isFocused)}
+            </div>
+            <div className={styles.moveButtons}>
+                <div className={styles.radioGroup}>
+                    <Tooltip
+                        placement="left"
+                        title={
+                            mode === ROTATE
+                                ? "Orthographic Camera"
```
Comment (meganrm) [inline src/components/CameraControls/index.tsx:225]: what's the reason for this ternary? it's giving the same title in each case?

File: src/assets/orthographic.svg
Comment (interim17) [reply src/assets/orthographic.svg:1]: fixed!

File: src/components/CameraControls/index.tsx (hunk 1 above)
Comment (interim17) [reply src/components/CameraControls/index.tsx:225]: that's a mistake, fixed

File: src/components/CameraControls/index.tsx [hunk 2]
```diff
@@ -31,9 +41,11 @@ const CameraControls = ({
     zoomOut,
     setPanningMode,
     setFocusMode,
+    setCameraType,
 }: CameraControlsProps): JSX.Element => {
     const [isFocused, saveFocusMode] = useState(true);
     const [mode, setMode] = useState(ROTATE);
+    const [camera, setCamera] = useState(PERSPECTIVE);
```
Comment (toloudis) [inline src/components/CameraControls/index.tsx:48]: a more descriptive name might be `cameraProjectionType`

Comment (toloudis) [reply src/components/CameraControls/index.tsx:48]: ...I see `setCameraType` was already used elsewhere, which is still better than `setCamera`.

**Review by toloudis — APPROVED** [review]

**Review by meganrm — APPROVED** [review]

**Pull request merged**